src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java

```diff
@@ -8,35 +8,39 @@
 /**
  * <body>
  * <p> This class is only meant for testing. </p>
- * <p> In html, closing all tags is not necessary. // violation
+ * <p> In html, closing all tags is not necessary. // violation 'Unclosed HTML tag found: p'
  * <li> neither is opening every tag <p> </li>
  * </body>
  *
  * @see "https://www.w3.org/TR/html51/syntax.html#optional-start-and-end-tags"
  */
 public class InputAbstractJavadocNonTightHtmlTags {
-    /** <p> <p> paraception </p> </p> */ // violation
+    /** <p> <p> paraception </p> </p> */ // violation 'Unclosed HTML tag found: p'
     private int field1;
 
     /**<li> paraTags should be opened</p> list isn't nested in parse tree </li>*/ // violation
+    // 'Unclosed HTML tag found: li'
     private int field2;
```

Comment:
Please use `// violation above 'message'` syntax.
We need violation trailing comment to be with message on same line.

Apply this to your changes.
Please keep violation comment outside of javadoc.

Reply:
Done

Reply:
Not done, please do for all your changes

Reply:
Sorry about that, I was having some issues with committing my changes. 
I've double-checked now and  I believe it is done.